beetsplug/bpd/__init__.py

```diff
@@ -1167,7 +1167,7 @@ def start_bpd(self, lib, host, port, password, volume, debug):
             server.run()
         except NoGstreamerError:
             global_log.error(u'Gstreamer Python bindings not found.')
-            global_log.error(u'Install "python-gst0.10", "py27-gst-python", '
+            global_log.error(u'Install "python-gst-1.0" '
```

Comment:
We should probably recommend the generic pygobject package (i.e., `python-gi` or `python2-gobject`) instead of a special-purpose python-gstreamer library.